script.js

```diff
@@ -308,7 +308,7 @@ function extractPrice(element) {
     if (dotIndex > -1 && dotIndex < comIndex) {
         newPrice = price.replace('.', '').replace(',', '.');
     } else {
-        newPrice = price.replace(',', '');
+        newPrice = price.replace(',', '.');
```

Comment:
@tomer953 Did you check it on multiple currencies?
If not, please check amazon.jp, co.uk, de, fr and others just to make sure this change is not breaking the behavior.

Reply:
I checked com, uk, it, de

i will update about jp tomorrow

Reply:
JP appears as "0", I don't think it related to this commit, but I'll try to fix it as well later today

![image](https://user-images.githubusercontent.com/1807493/107003103-eb928180-6794-11eb-8e0b-202e3b638eb9.png)

Reply:
OK found it, it is related to this commit,
since in JP and IT the comma is used differently, probably you already knew this :)
so hold on, I'll figure it out...

Reply:
"probably you already knew this :)"

yep 😆

Reply:
I think we must base on the 'host' (ie: co.uk \ .de \ .fr etc) since the currency itself is not enough to figure out the thousand seperator used in that country, for example, italy spain and france are all using Euro but each one of them using different thousand seperator (comma, dot, space)

so I wrote a little function to handle it, Ill give it a try and will update soon